### DISCUSSION TITLE
Is it possible to for whisper to remove accent on letters ? ("é,à,è ...)

### DISCUSSION THREAD

**Discussion by Pircla** [openai/whisper#2002] [Q&A]
Hello I am using whisper to transform audio into text, I use it in French and I get this kind of results :"j'hÃ©site Ã  l'acheter, je me
suis un peu informÃ© sur le jeu".
I would like to remove the accent, I mean I would like to have a result like this : "J'hesite a l'acheter, je me suis un peu informe sur le jeu".

Is it possible ? If yes how to do that ?

Thanks in advance

**Comment by EtienneAb3d** [comment]
@Pircla 
The result is UTF-8 char encoded. To get proper accents, you need to use an UTF-8 compatible tool.

**Reply by Pircla** [reply]
Thanks you ! I don't know it was that now it's working !
